src/engines/html5/media-source/adapters/native-adapter.js

```diff
@@ -704,15 +708,29 @@ export default class NativeAdapter extends BaseMediaSourceAdapter {
           } else if (settings.kind === 'captions' && this._config.enableCEA708Captions) {
             settings.label = settings.label || captionsTextTrackLabels.shift();
             settings.language = settings.language || captionsTextTrackLanguageCodes.shift();
+            settings.available = captionAvailable;
             parsedTracks.push(new PKTextTrack(settings));
             this._nativeTextTracksMap[settings.index] = textTracks[i];
+
+            if (!captionAvailable && !waitingFor708Captions) {
+              waitingFor708Captions = true;
+              this._maybeShow708Captions(textTracks[i]);
+            }
           }
         }
       }
     }
     return parsedTracks;
   }
 
+  _maybeShow708Captions(textTrack: any): void {
+    textTrack.mode = 'hidden';
+    this._eventManager.listenOnce(textTrack, 'cuechange', () => {
+      const textTracks = this._getPKTextTracks();
+      textTracks.forEach(track => (track.available = true) && (track.mode = 'disable'));
```

Comment:
if we have the language of the track as user selected will we still auto enable it?